.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+# Source files
+[*]
```

Comment:
Shouldn't we also specify the indentation here?

Reply:
This is the same one as enigma.go, I would like to have 2 chars space indentation, but currently we seem to be using tabs?